projects/angular/src/data/datagrid/utils/key-navigation-grid.controller.spec.ts

```diff
@@ -52,6 +52,10 @@ class TestComponent {
 
 export default function (): void {
   describe('Key navigation controller', function () {
+    function compareToActiveElement(element: HTMLElement): void {
```

Comment:
I would have named this `expectActiveElementToBe` because it's an expectation. I'd expect a "compare" method to return a boolean, not throw an error. Also, this name would keep make the test assertions easier to identify by using "expect" consistently.